### DISCUSSION TITLE
what the maximum length of prompt? also, what are proper guidelines of nature of prompt to get right results?

### DISCUSSION THREAD

**Discussion by sumitshrestha** [lllyasviel/Fooocus#1561] [General]
Here is the prompt i made. Foocus is not able to understand it and just creates gibberish image as a result.

`Setting: A grand, dimly lit hall in the heart of the medieval European Roman Inquisition court. Sunlight streams through stained glass windows, casting long shadows across the cobblestone floor.`

`Characters:`

`    Galileo Galilei: An elderly astronomer, dressed in simple robes, stands hunched before a towering Inquisitor. His face is etched with weariness and defiance, his eyes filled with a mix of sorrow and determination.
    Inquisitor: A stern-faced Cardinal, clad in crimson robes adorned with gold crosses. He holds aloft a celestial model depicting the Earth at the center of the universe (geocentrism), his expression a chilling blend of piety and ruthlessness.
    Crowd: A diverse group of onlookers line the walls, their faces illuminated by flickering candlelight. Among them are:
        Church officials: Cardinals, bishops, and priests, their eyes narrowed with suspicion and disapproval.
        Scholars and scientists: Some nod in agreement with the Inquisitor, others gaze at Galileo with empathy and concern.
        Common people: A mix of expressions – amusement at Galileo's humiliation, anger at the Inquisition's cruelty, disappointment in the silencing of truth, and a deep sense of hurt at the shattering of their worldview.`

`Mood: The scene should be charged with tension and drama. The air crackles with fear and uncertainty. Galileo's forced recantation hangs heavy in the air, a bitter testament to the power of dogma over reason.`

`Additional details:`

`    Include subtle symbolism in the background, such as astronomical charts, religious icons, and instruments of torture.
    Capture the play of light and shadow on the faces of the characters, emphasizing their emotional states.
    Use a color palette that is both rich and somber, reflecting the gravity of the situation.`

**Comment by botzero-net** [comment]
I'd suggest experimenting but if you want a "known working" solution I'd stick to SDXL prompt formatting.  Basic descriptions without helper words like "is" or "that", comma separated for context, parenthesis for weights, etc.  Even DALLE will take our long descriptions and stories and sort out the relevant bits for an image prompt that it feeds to its generator.
